cukinia/common_security_tests.d/hardening.conf

```diff
@@ -55,3 +55,11 @@ as "SEAPATH-00100 - /etc/securetty is empty" not cukinia_test -s /etc/securetty
 as "SEAPATH-00101 - PAM securetty module is active in 'login' policy" \
     cukinia_cmd grep -qE "^[^#].*pam_securetty.so" /etc/pam.d/login
 
+# Check that bootloader is password protected with a superuser "root" & one
+# unrestricted boot menuentry
+grep -q "^superusers=root" /boot/grub/grub.cfg && \
+grep -q "^password_pbkdf2 root grub.pbkdf2.sha512.65536.*$" /boot/grub/grub.cfg
+as "BP-028-R5-1 - grub root superuser is set in /boot/grub/grub.cfg" \
+	cukinia_test $? -eq 0
```

Comment:
for a nicer syntax use:
```
as "BP-028-R5-1 - grub root superuser is set in /boot/grub/grub.cfg" \
 cukinia_cmd grep -q "^superusers=root" /boot/grub/grub.cfg
as "BP-028-R5-3 - grub root superuser is password protected" \
 grep -q "^password_pbkdf2 root grub.pbkdf2.sha512.65536.*$" /boot/grub/grub.cfg
```

Reply:
can't use "^superusers=root$" ? To ensure that the superuser (which is password protected) is root and not root_lolz